network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/authenticated/ConversationApiV0.kt

```diff
@@ -370,7 +370,7 @@ internal open class ConversationApiV0 internal constructor(
         messageTimer: Long?
     ): NetworkResponse<EventContentDTO.Conversation.MessageTimerUpdate> =
         wrapKaliumResponse {
-            httpClient.put("$PATH_CONVERSATIONS/${conversationId.value}/$PATH_MESSAGE_TIMER") {
+            httpClient.put("$PATH_CONVERSATIONS/${conversationId.domain}/${conversationId.value}/$PATH_MESSAGE_TIMER") {
```

Comment:
the change log mentions that the nonqualified will be removed in v8 but checking swagger the qualified one exists since v0 so i just changed it to all versions